Modules/Sources/Storage/Bundle+Storage.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+
+extension Bundle {
+    static var storage: Bundle {
+#if DEBUG
+        // Workaround for https://forums.swift.org/t/swift-5-3-swiftpm-resources-in-tests-uses-wrong-bundle-path/37051
+        if let testBundlePath = ProcessInfo.processInfo.environment["XCTestBundlePath"],
+           let bundle = Bundle(path: "\(testBundlePath)/Modules_Storage.bundle") {
+            return bundle
+        }
+#endif
+        return Bundle.module
+
+    }
+}
```

Comment:
A necessary new file to allow accessing the Storage package bundle (only accessible via `Bundle.module`) outside of the package.